src/EventStore.Client.PersistentSubscriptions/PersistentSubscriptionStatus.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+
+namespace EventStore.Client
+{
+	/// <summary>
+	/// Represents the status of the persistent subscription.
+	/// </summary>
+	[Flags]
+	public enum PersistentSubscriptionStatus {
```

Comment:
This doesn't seem to be used except in one test (for a string comparison). Do we want to keep it, and is Flags necessarily appropriate? if so suggest renaming it to PersistentSubscriptionStatuses

Reply:
Removed it.